src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/client/OcculusTabRenderer.java

```diff
@@ -18,14 +18,13 @@
 import java.util.List;
 
 /**
- * Base class for all occulus tab renderers<br>
- * <p>To register a tab renderer you need to call <br>
- * {@code ArsMagicaAPI.get().registerOcculusTabRenderer(<registered occulus tab>, (occulusTab, player) -> <create your tab renderer>)}</p>
+ * Base class for all occulus tab renderers.
+ * To register a tab renderer, call {@code ArsMagicaAPI.get().registerOcculusTabRenderer(<registered occulus tab>, (occulusTab, player) -> <create your tab renderer>);}
```

Comment:
Have you checked the resulting javadoc? Does removing the html tags still look the same in the generated javadoc?

Reply:
It does.